graphqlapi/src/main/java/io/stargate/graphql/schema/fetchers/ddl/IndexFetcher.java

```diff
@@ -0,0 +1,50 @@
+package io.stargate.graphql.schema.fetchers.ddl;
+
+import static com.datastax.oss.driver.shaded.guava.common.base.Preconditions.checkNotNull;
+
+import graphql.schema.DataFetchingEnvironment;
+import io.stargate.auth.*;
+import io.stargate.auth.UnauthorizedException;
+import io.stargate.db.datastore.DataStoreFactory;
+import io.stargate.db.query.Query;
+import io.stargate.db.query.builder.QueryBuilder;
+import io.stargate.graphql.web.HttpAwareContext;
+
+public abstract class IndexFetcher extends DdlQueryFetcher {
+
+  protected Scope scope;
```

Comment:
Nit: if you're going to keep `getScope()`, you can turn this into a local variable.
Although I would personally remove `getScope()` and set it via the constructor instead.

Reply:
Done. 👍